docs/templates/plugin.rst.j2

```diff
@@ -117,9 +117,23 @@ Parameters
                 {% endfor %}
                 {# parameter name with required and/or introduced label #}
                 <td colspan="@{ from_kludge_ns('maxdepth') - loop.depth0 }@">
-                    <b>@{ key }@</b>
-                    {% if value.get('type', None) %}<br/><div style="font-size: small; color: red">@{ value.type }@</div>{% endif %}
-                    {% if value.get('required', False) %}<br/><div style="font-size: small; color: red">required</div>{% endif %}
+                    <b>@{ key }@</b></br>
+                    {# Clean up parameter types #}
+                    {% if value.type is not defined or value.type == 'str' %}
+                        {% set type = 'string' %}
```

Comment:
+1 to informing the reader what the default type is

Reply:
Well, this change stands on its own. The only problem is that the majority of the modules have incorrect documentation. So I have to fall back on the arg_spec. There's no practical way of fixing all the documentation errors. I started doing it, but I gave up. Fixing this once and for all is going to be a major improvement in contributor experience (and reviewing).